copter/source/index.rst

```diff
@@ -179,32 +179,18 @@ Getting more info
 .. toctree::
    :hidden:
    
-   Introducing Copter <docs/introduction>
+
+   Introduction to Copter<docs/copter-introduction>
    AutoPilot Hardware Options <docs/common-autopilots>
    First Time Setup <docs/initial-setup>
    First Flight and Tuning <docs/flying-arducopter>
-   If A Problem Arises <docs/common-when-problems-arise>
-   Flight Features <docs/copter-flight-features>
    Traditional Helicopters <docs/traditional-helicopters>
-   Tricopter <docs/tricopter>
-   SingleCopter and CoaxCopter <docs/singlecopter-and-coaxcopter>
-   HeliQuads <docs/heliquads>
-   SkyRocket <docs/skyrocket>
+   Ground Control Stations <docs/common-GCS>
+   If A Problem Arises <docs/common-when-problems-arise>
    Advanced Configuration <docs/common-advanced-configuration>
-   Mission Planning <docs/common-mission-planning>
```

Comment:
I think maybe Mission Planner and Logs should remain in this list 'cuz they are very commonly used.

Reply:
I can live with that

Reply:
however, after more thought, I think Logs should remain under If problems arise...while we use them a lot, many users do not until a problem does arise....and since they are automatic by default, no user setup is required usually